src/library/Uno.Material/Styles/Controls/v2/CalendarDatePicker.xaml

```diff
@@ -208,14 +208,14 @@
 							<TextBlock HorizontalAlignment="Stretch"
 									   VerticalAlignment="Top"
 									   Text="{TemplateBinding Header}"
-									   Style="{StaticResource M3MaterialBodySmall}"
+									   Style="{StaticResource BodySmall}"
```

Comment:
use the full style name here, alias should not be used here

Reply:
same in CalendarView + DatePicker too

Reply:
there is issue with "LabelLarge, LabelMedium" and probably all other typography style everywhere
use the script to check for references